library/src/test/java/org/apache/apex/malhar/lib/wal/FileSystemWALTest.java

```diff
@@ -53,20 +55,33 @@ private static Slice getRandomSlice(int len)
   {
     private String targetDir;
     FileSystemWAL fsWAL = new FileSystemWAL();
+    Configuration conf = new Configuration();
+    FileSystem fs;
 
     @Override
     protected void starting(Description description)
     {
-      TestUtils.deleteTargetTestClassFolder(description);
       targetDir = "target/" + description.getClassName() + "/" + description.getMethodName();
+
+      try {
```

Comment:
I suspect these changes are motivated by your local attempts to run the test against HDFS. If so, it may help others to document what else is needed for it. We cannot by default depend on HDFS but is is useful to know what other setup would be needed to manually test with it.